### ISSUE TITLE
When using Negotiated Rates for UPS an address is required for the shipper

### ISSUE THREAD

**Issue by jurgenkrol** [verbb/postie#82]
**Description**

When using Negotiated Rates for UPS an address is required for the shipper. The shipFrom address is correctly added, but this address is not added to the shipper. Only the account number is added.

**Steps to reproduce**
1. Enable Negotiated Rates for UPS
2. Put something in your cart
3. Goto the checkout page and fill in your address
4. In the next step the wrong rates are shown for UPS

**Additional info**
- Plugin version: 2.4.13
- Craft version: 3.7.26
- Multi-site: No

**Additional context**
This issue can be solved by adding the address to the shipper object below the shipper number, line 410 in verbb/postie/src/providers/UPS.php:

$shipper->setAddress($shipFromAddress);

**Comment by engram-design** [comment]
That makes sense, but I have no idea how negotiated rates have _been working_ up until this point. I'll look into it.
'

**Comment by jurgenkrol** [comment]
Thanks!
I think the solution might be simple. Just add the aforementioned line to UPS.php

**Comment by engram-design** [comment]
I've just added this for the next version. To get the fix early, change your `verbb/postie` requirement in `composer.json` to:

```
"require": {
  "verbb/postie": "dev-craft-3 as 2.4.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jurgenkrol** [comment]
Thanks for the fix!

**Comment by engram-design** [comment]
Fixed in [2.4.15](https://github.com/verbb/postie/releases/tag/2.4.15)
